pkgs/top-level/aliases.nix

```diff
@@ -1487,7 +1487,8 @@ mapAliases ({
   tex-gyre-pagella-math = throw "'tex-gyre-pagella-math' has been renamed to/replaced by 'tex-gyre-math.pagella'"; # Converted to throw 2022-02-22
   tex-gyre-schola-math = throw "'tex-gyre-schola-math' has been renamed to/replaced by 'tex-gyre-math.schola'"; # Converted to throw 2022-02-22
   tex-gyre-termes-math = throw "'tex-gyre-termes-math' has been renamed to/replaced by 'tex-gyre-math.termes'"; # Converted to throw 2022-02-22
-  textadept11 = textadept; # Added 2022-06-07
+  textadept11 = throw "textadept11 has been removed. Please use textadept12 instead"; # Added 2022-12-23 TODO: UPDATE THE DATE
+  textadept12 = textadept; # Added 2022-12-23 TODO: UPDATE THE DATE
```

Comment:
If we give up on keeping even the previous version, maybe _not_ add `textadept12` alias? `textadept11` alias was added when dropping the 10/11 two-versions support.

Reply:
Thank you for the feedback. So I should remove line 1491. What about line 1490 -- should I put that back the way it was?

Reply:
I think it's fine to replace it with a throw (as we will have no TA 11 afterwards) with advice to use `textadept` the non-version-pinned.

Reply:
done!